src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb

```diff
@@ -3337,12 +3337,12 @@ _Default:
         End Function
 
         Protected NotOverridable Overrides Function AnalyzeControlFlowCore(firstStatement As SyntaxNode, lastStatement As SyntaxNode) As ControlFlowAnalysis
-            Return Me.AnalyzeControlFlow(SafeCastArgument(Of StatementSyntax)(firstStatement, "firstStatement"),
-                                                   SafeCastArgument(Of StatementSyntax)(lastStatement, "lastStatement"))
+            Return Me.AnalyzeControlFlow(SafeCastArgument(Of StatementSyntax)(firstStatement, NameOf(firstStatement)),
+                                                   SafeCastArgument(Of StatementSyntax)(lastStatement, NameOf(firstStatement)))
```

Comment:
@AdamSpeight2008 @gafter This appears to have been accidentally changed from lastStatement to firstStatement.